extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -463,11 +463,11 @@ Adds support for Pinpoll. Configuration details can be found at [pinpoll.com](ht
 
 Type attribute value: `piStats`
 
-### PPAS analytics
+### Piwik PRO Analytics Suite
 
 Type attribute value: `ppasanalytics`
 
-Adds support for PPAS analytics. Configuration details can be found at [piwik.pro](https://developers.piwik.pro/).
+Adds support for Piwik PRO Analytics Suite. Configuration details can be found at [piwik.pro](https://developers.piwik.pro/integrations/AMP_integration.html).
```

Comment:
Is this page still under development? I see this when I click on it:
![image](https://user-images.githubusercontent.com/30711678/115239476-a2837580-a0d3-11eb-88c1-5ebd5f6f4813.png)

Reply:
You're right. I fixed the link.